Desktop/release/app/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "electron-react-boilerplate",
+  "name": "async-api-simulator",
```

Comment:
```suggesiton
  "name": "asyncapi-simulator",
```

Reply:
i dont understand when i got to the branch on my repo desktop/release/app/packages.json is correct. The pull request is not showing it correctly.